stubs/tqdm/tqdm/contrib/discord.pyi

```diff
@@ -8,10 +9,15 @@ from .utils_worker import MonoWorker
 __all__ = ["DiscordIO", "tqdm_discord", "tdrange", "tqdm", "trange"]
 
 class DiscordIO(MonoWorker):
+    API: str = "https://discord.com/api/v10"
+    UA: str = ...
     text: Incomplete
     message: Incomplete
```

Comment:
The `token`, `channel_id`, and `session` attibutes were also added in this release: https://github.com/tqdm/tqdm/compare/v4.66.6...v4.67.0#diff-f6ef645a2466cf282cfa55fb82f064281e15b1437c1dde109b172c7195e6ed4fR34

Reply:
fixed